src/Tools/dotnet-counters/Exporters/ConsoleWriter.cs

```diff
@@ -72,18 +72,27 @@ public ObservedTagSet(string tags)
         private string _errorText;
 
         private int _maxRow = -1;
-        private readonly bool _useAnsi;
 
         private int _consoleHeight = -1;
         private int _consoleWidth = -1;
+        private IConsole _console;
+        private bool oldCursorVisibility = true;
```

Comment:
Should this start with `_`?  I thought it was a local until I searched the file.

Reply:
yes, indeed it should